doc/sphinx-guides/source/admin/make-data-count.rst

```diff
@@ -0,0 +1,87 @@
+Make Data Count
+===============
+
+`Make Data Count`_ is a project to collect and standardize metrics on data use. They are part of a broader Research Data Alliance (RDA) `Data Usage Metrics Working Group`_ that they helped launch and they publish a `newsletter`_.
+
+.. _Make Data Count: https://makedatacount.org
+.. _Data Usage Metrics Working Group: https://www.rd-alliance.org/groups/data-usage-metrics-wg
+.. _newsletter: https://makedatacount.org/contact/
+
+.. contents:: Contents:
+	:local:
+
+Introduction
+------------
+
+All installations of Dataverse that use DOIs as persistent identifiers are encouraged to send data usage metrics to the "open hub" operated by DataCite for Make Data Count.
+
+Data repositories using Handles and other identifiers are not supported by Make Data Count but in the notes_ following a July 2018 webinar, you can see the project's response on this topic.
+
+.. _notes: https://docs.google.com/document/d/1b1itytDVDsI_Ni2LoxrG887YGt0zDc96tpyJEgBN9Q8/
+
+Make Data Count is built on top of existing standards such as COUNTER and SUSHI that come out of the article publishing world. To meet the needs of the data publishing world, Make Data Count created the `COUNTER Code of Practice for Research Data`_ (`preprint`_), which is the standard that Dataverse implements. The Make Data Count project has emphasized that they would like feedback on the code of practice.
+
+
+.. _COUNTER Code of Practice for Research Data: https://makedatacount.org/counter-code-of-practice-for-research-data/
+.. _preprint: https://doi.org/10.7287/peerj.preprints.26505v1
+
+Sending Metrics from Dataverse to the DataCite Hub
+--------------------------------------------------
+
+To configure Dataverse to send the metrics to the DataCite hub, you must set up a cron job to call the following API endpoint:
+
+``curl -X POST http://localhost:8080/api/admin/makeDataCount/sendToHub``
+
+The following metrics will be sent for each published dataset:
+
+- Views ("investigations" in COUNTER)
+- Downloads ("requests" in COUNTER)
+
+Retrieving Make Data Count Metrics from the DataCite Hub
+--------------------------------------------------------
+
+The following metrics can be downloaded directly from the DataCite hub (see https://support.datacite.org/docs/eventdata-guide) for datasets hosted by Dataverse installations that have been configured to send these metrics to the hub:
+
+- Total Views for a Dataset
+- Unique Views for a Dataset
+- Total Downloads for a Dataset
+- Downloads for a Dataset
+- Citations for a Dataset (via Crossref)
+
+Retrieving Make Data Count Metrics from Dataverse
+-------------------------------------------------
+
+Dataverse users might find it more convenient to retrieve Make Data Count metrics from their installation of Dataverse rather the DataCite hub.
```

Comment:
Interesting! So Dataverse would provide everything except citations in this case?